pkgs/development/tools/textql/default.nix

```diff
@@ -1,27 +1,32 @@
-{ lib, buildGoPackage, fetchFromGitHub, sqlite }:
+{ lib, buildGoModule, fetchFromGitHub, fetchpatch, sqlite }:
 
-buildGoPackage rec {
+buildGoModule rec {
   pname = "textql";
-  version = "2.0.3";
-
-  goPackagePath = "github.com/dinedal/textql";
+  version = "unstable-2021-07-06";
 
   src = fetchFromGitHub {
     owner  = "dinedal";
     repo   = "textql";
-    rev    = version;
-    sha256 = "1b61w4pc5gl7m12mphricihzq7ifnzwn0yyw3ypv0d0fj26h5hc3";
+    rev    = "fca00ecc76c8d9891b195ad2c1359d39f0213604";
+    sha256 = "1v1nq7q2jr7d7kimlbykmh9d73cw750ybcz7v7l091qxjsii3irm";
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/dinedal/textql/commit/a0d7038c8c30671dfd618f47322814ab492c11a1.patch";
```

Comment:
done

Reply:
> we should reference it via the fork 

Revert this change. If the fork is deleted the patch will disappear. Please add a comment linking the PR the commit came from.

Reply:
It'll be in the binary cache either way, so I don't really care, tbh.

The same diff can be gotten many ways, so how can we find a consensus here? I assume nixpkgs doesn't have a policy for this, despite there being dozens if not hundreds of occurrences of this throughout the codebase, as always?

Reply:
Using a source that won't disappear instead of one that may disappear seems link a better idea.

I've seen lots of reviews asking for a comment referencing the PR that a commit is sourced from, I haven't seen anyone ask that we actually use the PR authors fork to source the commits.